package.json

```diff
@@ -17,8 +17,8 @@
   "license": "MPL-2.0",
   "dependencies": {
     "mozilla-normandy-action-argument-schemas": "0.3.0",
-    "node-uuid": "1.4.8",
-    "nsp": "3.2.1"
+    "nsp": "3.2.1",
+    "uuid": "3.3.2"
```

Comment:
The reason the "order" changed was because I used `yarn remove node-uuid && yarn add uuid` instead of manually messing with this.